<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncompleteIndexDialog</class>
 <widget class="QWidget" name="IncompleteIndexDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Some of the folders you want to search in are not fully indexed yet or have been excluded from indexing. All the folders checked below will be scheduled for priority reindexing. If you uncheck a folder it may still be indexed later, unless it was manually excluded from indexing.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="3">
    <widget class="QListView" name="collectionView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="unselectAllBtn">
     <property name="text">
      <string>Unselect All</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="selectAllBtn">
     <property name="text">
      <string>&amp;Select All</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
